core/src/test/scala/org/apache/spark/scheduler/HealthTrackerSuite.scala

```diff
@@ -554,6 +554,49 @@ class HealthTrackerSuite extends SparkFunSuite with BeforeAndAfterEach with Mock
     verify(allocationClientMock).killExecutorsOnHost("hostA")
   }
 
+  test("excluding decommission and kills executors when enabled") {
+    val allocationClientMock = mock[ExecutorAllocationClient]
+    healthTracker = new HealthTracker(listenerBusMock, conf, Some(allocationClientMock), clock)
+
+    // verify we decommission when configured
+    conf.set(config.EXCLUDE_ON_FAILURE_KILL_ENABLED, true)
+    conf.set(config.DECOMMISSION_ENABLED.key, "true")
+    conf.set(config.EXCLUDE_ON_FAILURE_DECOMMISSION_ENABLED.key, "true")
+    healthTracker = new HealthTracker(listenerBusMock, conf, Some(allocationClientMock), clock)
+
+    // Fail 4 tasks in one task set on executor 1, so that executor gets excluded for the whole
```

Comment:
This seems to depend on our default configuration. Instead of assuming, can we use the default value of the confs directly instead?

Reply:
The default can be changed later in Apache Spark and can be different in the downstream, @holdenk .

Reply:
I've added explicit configs.